notebooks/official/feature_store/online_feature_serving_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    FILTER_COLUMNS = ["currency_code", "gender", "shipping_country_codes"]  # @param
</pre><p>Default value ["country"] instead of ["currency_code", "gender", "shipping_country_codes"]&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>